src/main/resources/translations/en-us.json

```diff
@@ -0,0 +1,6886 @@
+{
```

Comment:
Where does this file come from? Is it available in paper?

Reply:
Generated by metaminer (legal grayzone), let me make a pr for that actually